core/common/src/main/java/alluxio/cli/AbstractCommand.java

```diff
@@ -36,19 +36,7 @@ protected AbstractCommand() {}
    * @param args the arguments for the command, excluding the command name and options
    * @throws InvalidArgumentException when arguments are not valid
```

Comment:
add a return javadoc? What is the difference between returning `false` and throwing an exception?